cms/static/js/spec/views/modals/move_xblock_modal_spec.js

```diff
@@ -0,0 +1,73 @@
+define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
+        'common/js/spec_helpers/template_helpers', 'common/js/spec_helpers/view_helpers',
+        'js/views/modals/move_xblock_modal', 'js/models/xblock_info'],
+    function($, _, AjaxHelpers, TemplateHelpers, ViewHelpers, MoveXBlockModal, XBlockInfo) {
+        'use strict';
+        describe('MoveXBlockModal', function() {
+            var modal,
+                showModal,
+                DISPLAY_NAME = 'HTML 101',
+                OUTLINE_URL = '/course/cid?format=concise',
+                ANCESTORS_URL = '/xblock/USAGE_ID?fields=ancestorInfo';
+
+            showModal = function() {
+                modal = new MoveXBlockModal({
+                    sourceXBlockInfo: new XBlockInfo({
+                        id: 'USAGE_ID',
+                        display_name: DISPLAY_NAME,
+                        category: 'html'
+                    }),
+                    XBlockUrlRoot: '/xblock',
+                    outlineURL: OUTLINE_URL,
+                    XBlockAncestorInfoUrl: ANCESTORS_URL
+
+                });
+                modal.show();
+            };
+
+            beforeEach(function() {
+                setFixtures('<div id="page-notification"></div><div id="reader-feedback"></div>');
+                TemplateHelpers.installTemplates([
+                    'basic-modal',
+                    'modal-button',
+                    'move-xblock-modal'
+                ]);
+            });
+
+            afterEach(function() {
+                modal.hide();
+            });
+
+            it('rendered as expected', function() {
+                showModal();
+                expect(
+                    modal.$el.find('.modal-header .title').contents().get(0).nodeValue.trim()
+                ).toEqual('Move: ' + DISPLAY_NAME);
+                expect(
+                    modal.$el.find('.modal-sr-title').text().trim()
+                ).toEqual('Choose a location to move your component to');
+                expect(modal.$el.find('.modal-actions .action-primary.action-move').text()).toEqual('Move');
+            });
+
+            it('sends request to fetch course outline', function() {
```

Comment:
IMO there's no purpose in verifying that AJAX calls were made without also verifying the contract. If this is tested elsewhere then I'd just remove this test.